Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM node:10
```

Comment:
Is it possible to use `node-alpine` to shrink the size of the image?

Reply:
It's OK, But `node-alpine` maybe lost some  performance